scripts/releaser.sh

```diff
@@ -30,6 +30,14 @@ function generate_changelog {
   echo "## :package: Others"
   echo
   cat "${MERGED_PRS}" | grep -v "\!" | grep -v feat[:\(] | grep -v fix[:\(] | grep -v docs[:\(]
+
+  echo
+  echo "## :package: Docker Image"
+  echo
+  echo "\`\`\`sh"
+  echo "docker pull registry.k8s.io/external-dns/external-dns:${VERSION}"
```

Comment:
```suggestion
  echo "\`\`\`sh"
  echo "# This pull command only works when it's released
  echo "docker pull registry.k8s.io/external-dns/external-dns:${VERSION}"
```
@Raffo something like this ?

Reply:
makes sense to me.